### ISSUE TITLE
BT-03:form-type mapping table

### ISSUE THREAD

**Issue by odscjen** [open-contracting/european-union-support#176]
While going through guidance.yaml checking for examples that need to be expanded I've come across this inconsistency.

BT-03 guidance is currently

>  The codes are drawn from the [Form type](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/form-type) authority table in EU Vocabularies.
> Look-up the code in the [form type mapping table](codelists/form-type), add the values of the release tag column to `tag` and set `tender.status` to the value of the tender status column.

The eforms example is

> <cbc:NoticeTypeCode listName="competition">cn-standard</cbc:NoticeTypeCode>

But it's not the code inbetween the tags that's in the mapping table, it's the `listName` value. The code is BT-02 which we'd said to discard.

Suggest rewording the guidance as

>  The listName are drawn from the [Form type](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/form-type) authority table in EU Vocabularies.
> Look-up `cbc:NoticeTypeCode[@listName]` in the [form type mapping table](codelists/form-type), add the values of the release tag column to `tag` and set `tender.status` to the value of the tender status column.

**Comment by duncandewhurst** [comment]
Good spot. Your proposal looks good, with a couple of minor tweaks:

>  The values of `cbc:NoticeTypeCode[@listName]` are drawn from the [form type authority table](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/form-type) in the EU Vocabularies.
> Look up `cbc:NoticeTypeCode[@listName]` in the [form type mapping table](codelists/form-type), add the values of the release tag column to `tag` and set `tender.status` to the value of the tender status column.

**Comment by odscjen** [comment]
Reopening this following my test mapping of can_24_minimal.

The only way I had of looking up the guidance was via the XPATHs. This meant I pulled up BT-02 when looking for `cbc:NoticeTypeCode` and it says to discard. I remembered that there was a mapping for `cbc:NoticeTypeCode[@listName]` but it's likely that a normal user might miss this and not map this at all. 

Suggest we add to the BT-02 guidance to something like:

"Discard, and follow the guidance for BT-03 instead."

**Comment by jpmckinney** [comment]
Sounds good to add this clarification. However, mapping BT-03 is not really an alternative to mapping BT-02 - we just want to remind the implementer not to forget to map BT03. Maybe:

> Discard. Note that the `@listName` attribute of the XML element is mapped in [BT-03-notice](#BT-03-notice).
